todo_app/app.py

```diff
@@ -1,11 +1,23 @@
-from flask import Flask
+from flask import Flask , render_template, request
+from flask.helpers import url_for
+from werkzeug.utils import redirect
+from todo_app.data.session_items import get_items, add_item
 
 from todo_app.flask_config import Config
 
+
+
 app = Flask(__name__)
 app.config.from_object(Config())
 
+@app.route('/' )
+def home():
+    return render_template('index.html', get_items= get_items)
```

Comment:
Minor point, but at the moment the render_template is being passed `get_items` which is a function that returns a list of items. It would be more expected normally to return the list of items itself i.e
```suggestion
    return render_template('index.html', items=get_items())
```